specs/v1.yaml

```diff
@@ -1257,6 +1441,141 @@ components:
           example: direct
         address:
           $ref: "#/components/schemas/GrantAddress"
+    RecurringGrant:
+      type: object
+      description: |-
+        A RecurringGrant represents a successfully initiated recurring grant request with a Donor Advised Fund. RecurringGrants are created when a person interacts with an instance of Chariot Connect and successfully submits a recurring grant and completes the workflow. There can be many recurring grants associated with a Connect object and therefore a Nonprofit.
```

Comment:
it's worth explaining the relationship between a Recurring Grant a (regular) Grant here and what a user of the API should expect